src/aelplermagronen.md

```diff
@@ -19,19 +19,20 @@ Cooking time: ~30 minutes
 
 ## Directions
 
-1. Fry bacon cubes in pot (this pot will be used for everything so choose an appropriately large one)
+1. Fry bacon cubes in pot (this pot will be used for everything so choose an appropriately large one).
 2. Cut onions into half-rings and let them sweat in the same pot. Add some butter if your bacon was not fatty enough.
-3. Peel potatoes and cut them into ~1 cm/half inch cubes
+3. Peel potatoes and cut them into ~1 cm/half inch cubes.
 4. When the onions have become sufficiently cooked, add potatoes.
 5. Top everything with milk and let the potatoes cook for about 10 minutes.
 7. Add macaroni and the remaining milk to cover everything. Most of the milk will be absorbed by the macaroni.
 8. Shred your cheese.
-9. A minute or two before the macaroni are done, add the shredded cheese into the pot. It should appear a bit too runny in the pot. While cooling it will increase in viscosity quite a bit. If the final texture is not creamy enough it is most likely due to using the wrong cheese.
+9. A minute or two before the macaroni are done, add the shredded cheese into the pot. It should appear a bit too runny in the pot. While cooling it will increase in viscosity quite a bit. If the final texture is not creamy enough, it is most likely due to using the wrong cheese.
```

Comment:
I would suggest changing the `the macaroni are done` to `the macaroni is done` as it makes more grammatical sense

Reply:
IIRC, macaroni is also plural, but it does sound better using 'is' instead of 'are'. Thanks.